### PR TITLE
Add CTO Resources

### PR DESCRIPTION
**Pull request by mateusz-brainhub**
https://github.com/mateusz-brainhub/awesome-cto-resources

**[Resources to help you grow as a CTO like books, blog posts, lessons learned from tech leaders.]**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -463,6 +463,8 @@
 - [OKR Methodology](https://github.com/domenicosolazzo/awesome-okr) - Goal setting & communication best practices.
 - [Leading and Managing](https://github.com/LappleApple/awesome-leading-and-managing) - Leading people and being a manager in a technology company/environment.
 - [Indie](https://github.com/mezod/awesome-indie) - Independent developer businesses.
+- [CTO Resources](https://github.com/mateusz-brainhub/awesome-cto-resources) - Resources to help you grow as a CTO like books, blog posts, lessons learned from tech leaders.
```
Comment (legodude17) [inline readme.md]: I don't think the `like books, blog posts, lessons learned from tech leaders.` is necessary.

**Comment by legodude17** [comment]
The link in the PR description is broken.I would suggest using dashes instead of colons and putting the types in parenthesis. I suggest you add a LICENSE file as well.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/mateusz-brainhub/awesome-cto-resources
✖ Linting

  README.md:9:3
  ✖    1:1    Missing or invalid Table of Contents                          remark-lint:awesome/toc
  ✖    1:26   Invalid badge source                                          remark-lint:awesome/badge
  ✖    9:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖    9:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   10:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   10:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   11:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   11:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   12:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   12:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   13:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   13:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   14:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   14:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   15:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   15:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   16:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   16:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   17:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   17:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   18:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   18:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   19:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   19:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   20:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   20:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   21:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   21:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   22:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   22:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   23:3    Incorrect indentation before bullet: remove 2 spaces          remark-lint:list-item-bullet-indent
  ✖   23:5    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   28:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   28:177  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   29:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   29:3    Don’t pad link with inner spaces                              remark-lint:no-inline-padding
  ✖   29:127  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   30:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   30:115  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   31:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   31:192  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   32:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   32:92   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   33:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   33:188  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   34:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   34:69   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   39:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   39:83   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   40:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   40:145  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   41:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   41:106  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   42:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   42:125  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   43:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   43:154  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   44:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   44:232  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   45:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   45:175  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   46:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   46:113  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   47:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   47:137  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   48:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   48:186  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   49:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   49:116  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   50:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   50:131  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   51:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   51:207  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   52:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   52:163  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   53:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   53:151  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   54:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   54:157  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   55:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   55:123  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   60:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   60:103  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   61:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   61:3    Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖   62:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   62:173  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   63:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   63:129  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   64:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   64:148  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   69:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   69:79   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   74:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   74:85   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   75:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   75:70   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   80:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   80:79   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   81:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   81:197  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   82:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   82:118  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   83:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   83:161  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   88:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   88:204  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   89:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   89:131  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   94:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   94:100  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   95:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   95:180  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   96:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   96:139  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   97:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   97:377  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   98:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   98:276  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖   99:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖   99:178  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  100:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  100:106  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  105:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  105:3    Don’t pad link with inner spaces                              remark-lint:no-inline-padding
  ✖  106:99   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  107:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  107:120  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  108:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  108:123  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  109:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  109:54   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  114:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  114:127  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  115:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  115:150  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  120:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  120:138  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  126:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  126:160  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  127:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  127:171  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  132:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  132:137  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  133:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  133:86   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  134:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  134:56   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  139:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  139:105  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  140:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  140:186  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  141:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  141:159  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  146:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  146:192  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  147:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  147:152  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  152:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  152:171  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  153:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  153:104  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  154:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  154:172  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  155:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  155:99   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  156:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  156:124  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  157:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  157:64   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  158:1    Marker style should be *                                      remark-lint:unordered-list-marker-style
  ✖  158:142  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  177:1    License must be the last section                              remark-lint:awesome/license

  173 errors
```


https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**